src/main/java/com/spartronics4915/frc2019/subsystems/CargoChute.java

```diff
@@ -0,0 +1,216 @@
+package com.spartronics4915.frc2019.subsystems;
+
+import com.spartronics4915.frc2019.Constants;
+import com.spartronics4915.lib.drivers.TalonSRXFactory;
+import com.spartronics4915.lib.util.ILoop;
+import com.spartronics4915.lib.util.ILooper;
+import com.spartronics4915.lib.util.Logger;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.Solenoid;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class CargoChute extends Subsystem
+{
+
+    private static CargoChute mInstance = null;
+
+    public static CargoChute getInstance()
+    {
+        if (mInstance == null)
+        {
+            mInstance = new CargoChute();
+        }
+        return mInstance;
+    }
+
+    public enum WantedState // Each WantedState will correspond to a button
```

Comment:
What `WantedState` do we use if we just want the chute to hold the cargo at the top?